src/main/scala/temple/builder/DatabaseBuilder.scala

```diff
@@ -43,10 +49,66 @@ object DatabaseBuilder {
     ColumnDef(name, colType, typeConstraints ++ valueConstraints)
   }
 
+  def buildQuery(
+    serviceName: String,
+    service: ServiceBlock,
+    endpoints: Set[CRUD],
+    iDAttribute: IDAttribute,
+    createdByAttribute: CreatedByAttribute,
+  ): ListMap[CRUD, Statement] = {
+    val tableName = StringUtils.snakeCase(serviceName)
+    val columns   = service.primitiveAttributes.map { case (name, _) => Column(name) }.toSeq
```

Comment:
why don’t we use the foreign keys?

Reply:
I forgot we stored foreign keys as uuids